sdk-blueprints/python/run-tests.sh

```diff
@@ -20,7 +20,7 @@ while IFS= read -r dir; do
     error_flag=1
   fi
   cd - > /dev/null
-done < <(find "$SEARCH_PATH" -type d -name "visier-api*" -print)
+done < <(find "$SEARCH_PATH" -type d -name "visier_api*" -print)
```

Comment:
I assume this is an intentiional change?

Reply:
Yes. It is related to renaming packages.